src/containers/Projects/CreateProjectPopup/index.js

```diff
@@ -149,6 +149,7 @@ let CreateProjectPopup = (props) => {
             type="text"
             label="Enter Project Name (Up to 80 characters)"
             validate={[required, maxLength80]}
+            autoComplete="new-password"
```

Comment:
I don't think we need autoComplete feature for password for security purpose however not critical issue

Reply:
this is also being used as auto-complete=off  now as auto-complete=off is not working for chrome any more